Countries/ListViewController.swift

```diff
@@ -59,11 +59,7 @@ class ListViewController: UITableViewController {
     }
     
     override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-        guard let header = tableView.dequeueReusableHeaderFooterView(withIdentifier: CountryTableViewHeader.identifier) as? CountryTableViewHeader else { return UITableViewHeaderFooterView() }
-//        let earthImage = UIImage(named: TableConstants.headerEarthImageName)
-//        header.earthImageView.image = earthImage
-//        header.label.text = TableConstants.headerLabelText
-        return header
+        return tableView.dequeueReusableHeaderFooterView(withIdentifier: CountryTableViewHeader.identifier) ?? UITableViewHeaderFooterView()
```

Comment:
But you can return optional so it can be only
```
return tableView.dequeueReusableHeaderFooterView(withIdentifier: CountryTableViewHeader.identifier)
```